### DISCUSSION TITLE
Please clarify! Changing conversations

### DISCUSSION THREAD

**Discussion by TheEngineer71** [Soulter/hugging-chat-api#256] [Q&A]
I'm sorry, but I'm new to python. I'm just using this library in python3, and am wondering how to change the conversation. I am aware that there is an answer for this, and for that I really do apologise, I'm sorry for re-asking. When using the `conversation_list = chatbot.get_remote_conversations()` command in a .py script, and then print it, it outputs this: `[<hugchat.types.message.Conversation object at 0x103928e80>]`. how do I then use the `chatbot.change_conversation(id)` command to change it. When using the output as an id, it outputs this: `AttributeError: 'str' object has no attribute 'id'`. I've also tried it with an integer, such as id = 1, but it appears with this output: `AttributeError: 'int' object has no attribute 'id'`. Once again I really do apologise for asking for clarification for an already answered question.

**Comment by Occisor** [comment]
I also faced a misunderstanding of what to point out as `"id"`.
Managed to find out.
When you request `conversation_list = chatbot.get_remote_conversations()`, you get a list of chats, it can be divided into serial numbers, according to their conclusion. Next, you need to insert as follows:
`chatbot.change_conversation(conversation_list[2])`
In this example, we chose chat number 2.
